int16 senderID
uint64 localtime
uint64 distributedTime
uint64 lamportTime


